### ISSUE TITLE
No tag with id="pmx" found

### ISSUE THREAD

**Issue by kravchenkoloznia** [bavovanachte/sphinx-wavedrom#17]
Building pdf document from rst file with wavedrom diagram give error with specific wave.

Error:
```
Exception occurred:
  File "/usr/local/lib/python3.5/dist-packages/cairosvg/parser.py", line 414, in __init__
    'No tag with id="{}" found.'.format(element_id))
TypeError: No tag with id="pmx" found.
```

Wave example:

```
{ "signal": [
        { "name": "nibble", "wave": "xpx" }
]}
```

Build script:

```
sphinx-build -b latex -E -D wavedrom_html_jsinline=0 source/
```

As I understood that exception occurred only when you put "x" after "p" in wave (any amount of "." maybe added between "x" and "p" -- exception still there)

**Comment by wallento** [comment]
Thanks for reporting, this is an issue with wavedrompy and I will fix it asap.

**Comment by wallento** [comment]
This is fixed in wavedrom 1.8.0-post4 (see: https://github.com/wallento/wavedrompy/commit/766dec3e7dec99ccd1a5eb345e271861b80cca17)

Can you please give it a try and confirm? Thanks again for your report!

**Comment by bavovanachte** [comment]
Closing here as the issue was fixed in wavedrompy
